dotnet/xml/Microsoft.ML.Trainers/StochasticDualCoordinateAscentClassifier.xml

```diff
@@ -37,9 +37,9 @@
             False and <paramref name="NumThreads" /> to 1.
             Elastic net regularization can be specified by the <paramref name="L2Const" /> and <paramref name="L1Threshold" /> parameters. Note that the <paramref name="L2Const" /> has an effect on the rate of convergence. 
             In general, the larger the <paramref name="L2Const" />, the faster SDCA converges.
-            </para><a href="https://www.microsoft.com/en-us/research/wp-content/uploads/2016/06/main-3.pdf">Scaling Up Stochastic Dual Coordinate Ascent</a>.
-            <a href="http://www.jmlr.org/papers/volume14/shalev-shwartz13a/shalev-shwartz13a.pdf">Stochastic Dual Coordinate Ascent Methods for Regularized Loss Minimization</a>.
-            </remarks>
+            </para><para>For more information see:<br/><a href="https://www.microsoft.com/en-us/research/wp-content/uploads/2016/06/main-3.pdf">Scaling Up Stochastic Dual Coordinate Ascent</a><br/>
```

Comment:
nit: missing comma after for more information

Reply:
Missed this.